src/main/java/com/iexec/common/utils/FeignBuilder.java

```diff
@@ -82,7 +84,8 @@ public Object decode(Response response, Type type) throws IOException {
                 })
                 .logger(new Slf4jLogger())
                 .logLevel(logLevel)
-                .requestInterceptor(template -> template.header("Content-Type", "application/json"));
+                .requestInterceptor(template -> template.header("Content-Type", "application/json"))
```

Comment:
What do you think about starting with:
`public static Feign.Builder createBuilder(Logger.Level logLevel, boolean retrier)`